### PR TITLE
update platform/bundles explanation and update deploying by hand section

### PR DESCRIPTION
**Pull request by xsalefter**
Fix this issue: https://github.com/killbill/killbill-docs/issues/463

### CODE REVIEW

File: userguide/tutorials/plugin_installation.adoc [hunk 1]
```diff
@@ -28,9 +29,9 @@ Under `root`, you will find the following:
 
 * A `sha1.yml` file which is a used by the `KPM` tool to keep track of artifacts that were already downloaded to avoid downloading binaries already present on the filesystem. KPM also offers the `--force-download` to override that behavior.
 
-* A `platform` folder which contains the `kpm` OSGi bundle as well as a set of other OSGI bundles required for things like OSGI logging, OSGI console, ....
+* A `platform` folder: Also known as *bundles*. This is a directory for deploying Kill Bill's internal OSGI bundles, such as KPM, Logging, Eureka, Metrics, and any JARs/OSGI plugins from `<killbill-platform>/osgi-bundles`. Deploying 3rd party plugins into this directory is not recommended.
 
-* A `plugins` folder which contains: 
+* A `plugins` folder: This is where your plugin or 3rd party plugins deployed. This folder contains:
```
Comment (reshmabidikar) [inline userguide/tutorials/plugin_installation.adoc]:
```suggestion
* A `plugins` folder: This is where your plugin or 3rd party plugins are deployed. This folder contains:
```

File: userguide/tutorials/plugin_installation.adoc [hunk 2]
```diff
@@ -167,7 +168,25 @@ curl -v \
 
 === Deploying by Hand
 
-Deploying by hand consists of building the self contained OSGI jar, and copying that jar at the right location. For example, the `adyen` plugin with a version with version `0.3.2` would show up as the following:
+As explained in link:#_overview[overview section], Kill Bill have 2 locations to deploying plugins:
```
Comment (reshmabidikar) [inline userguide/tutorials/plugin_installation.adoc]:
```suggestion
As explained in link:#_overview[overview section], Kill Bill has two locations to deploying plugins:
```

File: userguide/tutorials/plugin_installation.adoc [hunk 3]
```diff
@@ -167,7 +168,25 @@ curl -v \
...
+
+1. `/platform` (bundles) for Kill Bill internal plugins, and;
+2. `/plugins` for 3rd party plugins.
+
+Deploying bundles is straightforward:
+
+1. Build `killbill-platform` project.
```
Comment (reshmabidikar) [inline userguide/tutorials/plugin_installation.adoc]: Does this refer to the https://github.com/killbill/killbill-platform repo? If so, consider adding a link to the repo to make this clear.

File: userguide/tutorials/plugin_installation.adoc [hunk 4]
```diff
@@ -167,7 +168,25 @@ curl -v \
...
+2. Copy desired bundles to `/platform` folder.
+3. Start Kill Bill server.
+
+If you want to deploy 3rd party or your own plugins, you need to:
+
+1. Create new folder with the name that represents your plugin. It is a good practice to set plugin name with "-plugin" suffix.
```
Comment (reshmabidikar) [inline userguide/tutorials/plugin_installation.adoc]:
```suggestion
1. Create a new folder with the name that represents your plugin. It is a good practice to set plugin name with "-plugin" suffix.
```

File: userguide/tutorials/plugin_installation.adoc [hunk 5]
```diff
@@ -167,7 +168,25 @@ curl -v \
...
+2. Add sub-folder that represent version of your plugin. For example, you can set `1.0.0` if it is brand-new plugin.
```
Comment (reshmabidikar) [inline userguide/tutorials/plugin_installation.adoc]:
```suggestion
2. Add a sub-folder that represents your plugin version.
```

File: userguide/tutorials/plugin_installation.adoc [hunk 6]
```diff
@@ -167,7 +168,25 @@ curl -v \
...
+3. Put your plugin JAR file into folder you've created in point 2. It is recommended that your plugin have this following format: `<plugin-name>-<semver-plugin-version>.jar`.
```
Comment (reshmabidikar) [inline userguide/tutorials/plugin_installation.adoc]:
```suggestion
3. Copy your plugin JAR file into the folder you've created in step 2. It is recommended that your plugin jar is named in the following format: `<plugin-name>-<semver-plugin-version>.jar`.
```

**Review by reshmabidikar — COMMENTED** [review]
Looks good, I've left a few minor comments.

File: userguide/tutorials/plugin_installation.adoc (hunk 3 above)
Comment (xsalefter) [reply userguide/tutorials/plugin_installation.adoc]: fixed

**Review by reshmabidikar — APPROVED** [review]

**Pull request merged**